ui/src/app/shared/utils.ts

```diff
@@ -136,7 +136,7 @@ export const Utils = {
         resourceVersion?: string;
     }) {
         const queryParams: string[] = [];
-        const fieldSelector = this.fieldSelectorParams(filter.namespace, filter.name, filter.createdAfter, filter.finishedBefore);
+        const fieldSelector = this.fieldSelectorParams(filter.namespace, filter.name, filter.createdAfter, filter.finishedBefore, filter.namePrefix, filter.namePattern);
```

Comment:
hmm the last two are not exactly field selectors (they are not valid k8s field selectors)